source/validate_instruction.cpp

```diff
@@ -340,9 +341,44 @@ spv_result_t RegisterDecorations(ValidationState_t& _,
   return SPV_SUCCESS;
 }
 
+// Parses OpExtension instruction and registers extension.
```

Comment:
Should say what the return code means. ?  Especially since it's always SPV_SUCCESS